tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -401,7 +402,15 @@ bool isFusible(Operation* op) {
 // For some ops (e.g. lmhlo ops), some operands are the output memrefs
 // Thus these operands are supposed to be updated.
 int getNumResultOperands(Operation* op) {
-  if (op->getDialect()->getTypeID() != TypeID::get<lmhlo::LmhloDialect>()) {
+  if (auto customOp = dyn_cast_or_null<lmhlo_disc::CustomCallOp>(op)) {
```

Comment:
What is the scenario of `lmhlo_disc::CustomCallOp`

Reply:
For example, random_uniform, topk, quantized_conv ...